massgen/backend/antigravity_cli.py

```
@@ -887,15 +1208,35 @@ async def _tail() -> None:
+            watch_stop.set()
+            watcher.cancel()
             await self._terminate_subprocess(proc)
             yield StreamChunk(type="error", error=str(exc))
             return
+        finally:
+            watch_stop.set()
+            watcher.cancel()
+            try:
+                await watcher
+            except (asyncio.CancelledError, Exception):
+                pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)